src/sqs.ts

```diff
@@ -58,6 +64,17 @@ export type SQSPartialBatchResponse = {
   }[];
 };
 
+const safeRedactor =
+  (logger: LoggerInterface, redactor: (body: string) => string) =>
+  (body: string) => {
+    try {
+      return redactor(body);
+    } catch (error) {
+      logger.error({ error, body }, 'Failed to redact message body');
```

Comment:
If this did in fact include PHI and someone wanted to redact, but did something wrong, we'll still end up logging the PHI. Does it make sense to just not log the body here?